modules/builds-understanding-openshift-pipeline.adoc

```diff
@@ -14,20 +14,20 @@ Jenkins images on {product-title} are fully supported and users should follow Je
 
 Pipelines give you control over building, deploying, and promoting your applications on {product-title}. Using a combination of the Jenkins Pipeline build strategy, `jenkinsfiles`, and the {product-title} Domain Specific Language (DSL) provided by the Jenkins Client Plug-in, you can create advanced build, test, deploy, and promote pipelines for any scenario.
 
-*{product-title} Jenkins Sync Plugin*
+*{product-title} Jenkins Sync Plug-in*
 
-The {product-title} Jenkins Sync Plugin keeps the build configuration and build objects in sync with Jenkins jobs and builds, and provides the following:
+The {product-title} Jenkins Sync Plug-in keeps the build configuration and build objects in sync with Jenkins jobs and builds, and provides the following:
 
  * Dynamic job and run creation in Jenkins.
  * Dynamic creation of agent pod templates from image streams, image stream tags, or config maps.
  * Injecting of environment variables.
  * Pipeline visualization in the {product-title} web console.
- * Integration with the Jenkins git plugin, which passes commit information from
- * Synchronizing secrets into Jenkins credential entries {product-title} builds to the Jenkins git plugin.
+ * Integration with the Jenkins git plug-in, which passes commit information from
```

Comment:
> @bmcelvee @rolfedh this point seems incomplete. Tagging you Brandi, in case you have any inputs, since you handed over this content to us.

Thanks, I've created https://issues.redhat.com/browse/RHDEVDOCS-3953 to fix this bug in a separate PR.